site/public/css/server.css

```diff
@@ -2029,3 +2029,15 @@ header > * {
 .mermaid-element {
     width: 100%;
 }
+
+.custom-scrollbar::-webkit-scrollbar {
```

Comment:
Have you tested these to make sure they work on all browsers?  Is there any reason why these are necessary instead of just relying on native OS scrollbars?  Scrollbars are tricky because they look, and act, differently on different systems and browsers.

Reply:
It seems to work for both Chrome and Firefox, I am only using it because I hate the way the native windows scrollbar looks.

Reply:
Ok, I'm still a little wary of it but it does seem to work as expected on safari and chrome.  It does look a little weird in light mode though.  You might want to have the background color update with the theme.